spark2/src/test/java/org/apache/iceberg/spark/source/TestFilteredScan.java

```diff
@@ -469,6 +473,21 @@ public void testPartitionedByDataStartsWithFilter() {
     Assert.assertEquals(1, reader.planInputPartitions().size());
   }
 
+  @Test
+  public void testPartitionedByDataNotStartsWithFilter() {
+    File location = buildPartitionedTable("partitioned_by_data", PARTITION_BY_DATA, "data_ident", "data");
+
+    DataSourceOptions options = new DataSourceOptions(ImmutableMap.of(
+        "path", location.toString())
+    );
```

Comment:
It seems weird to have one `)` on the same line and a second on its own line.

Reply:
Agreed.